Description: Ignore the shadow plugin
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -73,5 +73,3 @@
     gradleVersion = '2.3'
 }
 
-//Task used by Heroku for staging
-task stage(dependsOn: [':json-path-web-test:clean', 'json-path-web-test:shadowJar']) {}
--- a/json-path-web-test/build.gradle
+++ b/json-path-web-test/build.gradle
@@ -1,4 +1,3 @@
-apply plugin: 'com.github.johnrengelman.shadow'
 apply plugin: 'application'
 
 displayName = "JsonPath Test Bench"
