Description: Fixes the compatibility with Gradle 3.4+
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/json-path/build.gradle
+++ b/json-path/build.gradle
@@ -1,8 +1,6 @@
 apply from: "$rootDir/gradle/publishMaven.gradle"
 
 
-displayName = "Json Path"
-
 description = "Java port of Stefan Goessner JsonPath."
 
 jar {
--- a/json-path-assert/build.gradle
+++ b/json-path-assert/build.gradle
@@ -1,7 +1,5 @@
 apply from: "$rootDir/gradle/publishMaven.gradle"
 
-displayName = "JsonPath Assert"
-
 description = "Assertions on Json using JsonPath"
 
 jar {
--- a/json-path-web-test/build.gradle
+++ b/json-path-web-test/build.gradle
@@ -1,7 +1,5 @@
 apply plugin: 'application'
 
-displayName = "JsonPath Test Bench"
-
 description = "Web app that compares different JsonPath implementations."
 
 mainClassName = 'com.jayway.jsonpath.web.boot.Main'
